docs/config-file-curations-yml.md

```diff
@@ -20,6 +20,9 @@ Curations can be used to:
   * metadata-only packages, such as Maven BOM files, do not have any source code. Thus, when the flag is set the
   _downloader_ just skips the download and the _scanner_ skips the scan. Also, any _evaluator rule_ may optionally skip
   its execution.
+* set the _is_modified_ flag:
+  * indicates whether the source code of this package has been modified compared to the original source code, e.g., in
```

Comment:
nit: Would it make sense to speak of 'files' rather than 'source code' ? It can apply to any file, like data files as well, e.g. https://github.com/alex/certifi/blob/master/certifi/cacert.pem.